receiver/hazelcast.yaml

```
@@ -0,0 +1,7 @@
+hazelcast:
+  network:
+    join:
+      multicast:
+        enabled: false
+      kubernetes:
+        enabled: true
```

```suggestion
      kubernetes:
        enabled: true
```
Starting from 4.1 this should be enough.

Could you also try if it works if you remove the configuration at all? In 4.1 we added auto-detection feature and this configuration should not be needed.

---

Tested and it works, will chance accordingly in other guides